### PR TITLE
Add post method & display comments on screen

### PR DESCRIPTION
**Pull request by mollygraton**
This PR is for step 4 of week three, which allows users to enter comments through a form. These comments are displayed on the web page.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,17 +27,25 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> messages = new ArrayList<String>(Arrays.asList(
-      "Welcome to my page!", "Porfolio time!", "This is my portfolio"
-  ));
+  private ArrayList<String> comments = new ArrayList<String>(Arrays.asList(""));
```
Comment (cisxingg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: In this case you can probably remove 'Arrays.asList("")' as you are just creating an empty ArrayList :)

**Review by cisxingg — APPROVED** [review]
LG, one comment :)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -27,17 +32,40 @@
...
 
-  private ArrayList<String> messages = new ArrayList<String>(Arrays.asList(
-      "Welcome to my page!", "Porfolio time!", "This is my portfolio"
-  ));
+  private ArrayList<String> comments = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(messages);
+    Query query = new Query("Input");
```
Comment (warfa-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:39]: make this a file constant

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -27,17 +32,40 @@
...
+        comments.add((String) entity.getProperty("content"));
+    }
+
+    String json = convertToJson(comments);
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException  {
```
Comment (warfa-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:54]: In general, hard coded values aren't good. Make these all constants in this file or in a package wide constants class

**Review by warfa-google — APPROVED** [review]
LGTM. As we spoke about, add the constant file in a new PR

**Pull request merged**